kubectl-login.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Probably we'll need to rename this to something else, as if we put both kubectl-login and kubectl-login.sh on PATH, there will be a slight name clash.